contracts/libraries/SemverLibrary.sol

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+library SemverLibrary {
```

Comment:
Can we make this simpler by assuming that semver is always x.y.z where numbers are 0 to 255?

Reply:
no